runtime/Cargo.toml

```diff
@@ -51,6 +51,10 @@ pallet-contracts = { git = "https://github.com/paritytech/substrate", package =
 pallet-contracts-primitives = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-primitives", default-features = false , branch = "polkadot-v0.9.29" }
 pallet-contracts-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc-runtime-api", default-features = false , branch = "polkadot-v0.9.29" }
 
+# Chain extension
+pallet-assets = { git = "https://github.com/paritytech/substrate", package = "pallet-assets", default-features = false, branch = "polkadot-v0.9.29" }
```

Comment:
Can you move that up to the other pallets? This is not a chain extension per se.